qiskit/transpiler/passes/routing/stochastic_swap.py

```diff
@@ -324,7 +344,9 @@ def _mapper(self, circuit_graph, coupling_graph, trials=20):
                             best_circuit,
                         )
 
-            else:
+            elif not cf_layer and not (
+                subdag.depth() == 1 and subdag.op_nodes()[0].name == "continue_loop"
+            ):
```

Comment:
`continue` had to be handled a little differently than other control flow instructions since it doesn't subclass ControlFlowOp. If we're not handling `continue` at this time, however, it can be removed here and resolved later.